src/graph_compute/bsp_scheduling.h

```diff
@@ -14,14 +14,18 @@ static void init_scheduling(vertex_t * const nodes, const vid_t cntNodes,
   // no-op
 }
 
-static void execute_round(const int round, vertex_t * const nodes, const vid_t cntNodes,
+static void execute_round(const int numRounds, vertex_t * const nodes,
```

Comment:
So, does this not imply that I would have to merge in these changes to my current branch?  That seems like a pain in the ass.  Of course, if we serialize the process: branch, do work, get feedback & address it, commit to master, repeat.  This seems nice and neat, however, I'm in an insane hurry to get this working and serializing worries me...